tests/cql_query_test.cc

```diff
@@ -4261,3 +4261,135 @@ SEASTAR_TEST_CASE(test_rf_expand) {
+        std::ostringstream ss;
+
+        schema->describe(ss);
+        BOOST_CHECK_EQUAL(ss.str(), "CREATE  TABLE ks.cf (\n"
+                "    pk blob,\n"
+                "    col1 blob,\n"
+                "    col2 blob,\n"
+                "    PRIMARY KEY (pk)\n"
+                ") WITH bloom_filter_fp_chance = 0.01\n"
+                "    AND caching = {'keys': 'ALL','rows_per_partition': 'ALL'}\n"
+                "    AND comment = ''\n"
+                "    AND compaction = {'class': 'SizeTieredCompactionStrategy'}\n"
+                "    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}\n"
+                "    AND crc_check_chance = 1\n"
+                "    AND dclocal_read_repair_chance = 0.1\n"
+                "    AND default_time_to_live = 0\n"
+                "    AND gc_grace_seconds = 864000\n"
+                "    AND max_index_interval = 2048\n"
+                "    AND memtable_flush_period_in_ms = 0\n"
+                "    AND min_index_interval = 128\n"
+                "    AND read_repair_chance = 0\n"
+                "    AND speculative_retry = '99.0PERCENTILE';\n");
+        e.execute_cql("DROP TABLE ks.cf").get();
+        e.execute_cql(ss.str()).get();
+        auto new_schema = e.local_db().find_schema("ks", "cf");
+
+        cmp_schema(schema, new_schema);
+
+        e.execute_cql("CREATE  TABLE ks.cf1 (\n"
+                "    pk blob,\n"
+                "    ck blob,\n"
+                "    col1 blob,\n"
+                "    col2 blob,\n"
+                "    PRIMARY KEY (pk, ck))").get();
+        schema = e.local_db().find_schema("ks", "cf1");
+
+        std::ostringstream ss1;
+        schema->describe(ss1);
+        BOOST_CHECK_EQUAL(ss1.str(), "CREATE  TABLE ks.cf1 (\n"
+                "    pk blob,\n"
+                "    ck blob,\n"
+                "    col1 blob,\n"
+                "    col2 blob,\n"
+                "    PRIMARY KEY (pk, ck)\n"
+                ") WITH CLUSTERING ORDER BY (ck ASC)\n"
+                "    AND bloom_filter_fp_chance = 0.01\n"
+                "    AND caching = {'keys': 'ALL','rows_per_partition': 'ALL'}\n"
+                "    AND comment = ''\n"
+                "    AND compaction = {'class': 'SizeTieredCompactionStrategy'}\n"
+                "    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}\n"
+                "    AND crc_check_chance = 1\n"
+                "    AND dclocal_read_repair_chance = 0.1\n"
+                "    AND default_time_to_live = 0\n"
+                "    AND gc_grace_seconds = 864000\n"
+                "    AND max_index_interval = 2048\n"
+                "    AND memtable_flush_period_in_ms = 0\n"
+                "    AND min_index_interval = 128\n"
+                "    AND read_repair_chance = 0\n"
+                "    AND speculative_retry = '99.0PERCENTILE';\n");
```

Comment:
What about comparing the schemas with `cmp_schema` for this case?

Also, I think this big description strings should be moved to a string variable and used both during table creation and here. Reasons for that:
1. You start by creating the table without specifying all these configuration variables like `bloom_filter_fp_chance`, and then compare the results *with* these values - which means that every single change in the default value will also unnecessarily invalidate this test case.
2. It will simply result in less code duplication.

Reply:
@psarna I've forced push a new version, switching to a single string to create a table and to compare the generated schema to remove some of the tests.

Because  it's the same string, we know it works (or there wouldn't be a schema)